lib/OpenQA/WebAPI/Plugin/ObsRsync/templates/ObsRsync_index.html.ep

```diff
@@ -6,11 +6,43 @@
 
 % title 'OBS synchronization folders';
 <h2><%= 'OBS synchronization folders ' %></h2>
-<ul>
-% for my $folder (@$folders) {
-    <li><%= link_to $folder => url_for('plugin_obs_rsync_folder', folder => $folder) %></li>
+
+<div class="row">
+    <div class="col-sm-12">
+
+        %= include 'layouts/info'
+
+        <table class="table table-striped">
+            <thead>
+                <tr>
+                    <th>Project</th>
+                    <th>Last sync</th>
+                    <th>Sync status</th>
+                    <th>Job created</th>
+                    <th>Dirty status</th>
+                </tr>
+            </thead>
+            <tbody>
+            % for my $folder (sort keys %$folders_info) {
+                % my $folder_ident = $folder;
+                % $folder_ident =~ s/\W//g; # remove all special characters
+                % my $folder_info = $folders_info->{$folder};
+                <tr id="folder_<%= $folder_ident %>">
+                    <td class="project"> <%= link_to $folder => url_for('plugin_obs_rsync_folder', folder => $folder) %> </td>
```

Comment:
Why adding additinoal white spaces before and after the actual content of these cells?